app/plugin.go

```diff
@@ -547,3 +569,15 @@ func removePK(publicKeys []string, filename string) []string {
 	}
 	return publicKeys
 }
+
+func verifyFileSignature(signedFile, publicKey, signature io.Reader) error {
```

Comment:
since this is in the `app` namespace, should this be named plugin specific? `verifyPluginSignature`

Reply:
Well doesn't seem to have anything specific to plugin at this point.